guides/V1_wwsg_Webhooks.md

```diff
@@ -47,6 +47,8 @@ Available events are:
 
 **reaction_removed**: each time a reaction is removed from a message (API is EXPERIMENTAL)
 
+_Since this is an_ `EXPERIMENTAL` _capability, complete information can be found in our github repo, [Reactions Overview](./guides/V1_wwsg_Reactions.md), [Adding Reactions](./guides/V1_Add_Reaction.md), [Removing Reactions](./guides/V1_Remove_Reaction.md), and [Query for who's Reacting](./guides/V1_Reacting_Users.md)_.
```

Comment:
This change is not really necessary as this MD file gets overwritten at build time based on the yml counterpart.